filecoin-proofs/src/api/internal.rs

```diff
@@ -192,7 +196,10 @@ fn get_post_verifying_key(
     };
 
     Ok(lookup_verifying_key(
-        format!("POST[{}]", usize::from(sector_bytes)),
+        format!(
+            "POST[{}]",
```

Comment:
Is it true that this key will need to be updated to include PoSt partitions?

Reply:
Yes, verifying keys will need to be fetched by both sector size and number of partitions.